c7n/resources/emr.py

```diff
@@ -284,3 +285,46 @@ def query(self):
             value = [self.value]
 
         return {'Name': self.key, 'Values': value}
+
+
+@resources.register('emr-security-configuration')
+class EMRSecurityConfiguration(QueryResourceManager):
+    """Resource manager for EMR Security Configuration
+    """
+
+    class resource_type(TypeInfo):
+        service = 'emr'
+        arn_type = 'emr'
+        permission_prefix = 'elasticmapreduce'
+        enum_spec = ('list_security_configurations', 'SecurityConfigurations', None)
+        detail_spec = ('describe_security_configuration', 'Name', 'Name', None)
+        id = name = 'Name'
+
+    permissions = ('elasticmapreduce:ListSecurityConfigurations',
+                  'elasticmapreduce:DescribeSecurityConfiguration',)
+
+    def augment(self, resources):
+
+        def _augment(r):
+            # Convert the json string to dict
+            r['SecurityConfiguration'] = json.loads(r['SecurityConfiguration'])
+            return r
+
+        # Describe notebook-instance & then list tags
+        resources = super().augment(resources)
+        return list(map(_augment, resources))
```

Comment:
this seems like it would simpler/more readable as a for loop for deserializing the json

Reply:
Totally Agree, made the change